service/config.go

```diff
@@ -74,3 +74,9 @@ type CRIKeychainConfig struct {
 
 // ResolverConfig is config for resolving registries.
 type ResolverConfig resolver.Config
+
+// SnapshotterConfig is snapshotter-related config.
+type SnapshotterConfig struct {
+	// MinLayerSize skips remote mounting of smaller layers
+	MinLayerSize int `toml:"min_layer_size"`
```

Comment:
we should change the runtime `MinLayerSize` to `int64` to keep it consistent with 

https://github.com/awslabs/soci-snapshotter/blob/56534c70965d326ed9963f9a0fa5f0ecba9d3bb9/soci/soci_index.go#L217